tck/src/main/java/org/eclipse/microprofile/openapi/apps/airlines/resources/ZepplinResource.java

```diff
@@ -36,7 +38,29 @@ public class ZepplinResource {
     @Operation(summary = "Deprecate outdated airship technology", operationId = "deprecateZepplin")
     @Produces("text/plain")
     @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
-    public Response deprecateZepplin() {
+    public Response deprecateZepplin(String string) {
+        return Response.ok().build();
+    }
+
+    @HEAD
+    @Path("{id}")
+    @APIResponse(responseCode = "200", description = "Review deleted")
+    @APIResponse(responseCode = "404", description = "Review not found")
+    @Operation(summary = "Deprecate outdated airship technology", operationId = "deprecateZepplin")
+    @Produces("text/plain")
+    @SecurityRequirement(name = "mutualTLSScheme", scopes = "zepplinScope")
+    public Response headZepplin(String string) {
+        return Response.ok().build();
+    }
```

Comment:
The changes in this file don't affect the tests any more, I guess you left them in because you've used them in other PRs that build on this one?

That's ok, but the `@GET` and `@HEAD` methods still have the `string` entity parameter and this should be removed.